liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -765,6 +765,21 @@ private void addDomPropertyIfNotFound(Xpp3Dom sysProps, String key, String value
             sysProps.addChild(element(name(key), value).toDom());
         }
     }
+    
+    private Element getBoostrapProps () {
+    	Element retVal = null;
+    	Plugin plugin = getPlugin("io.openliberty.tools", "liberty-maven-plugin");
+    	Xpp3Dom config = getPluginConfig(plugin, ""); // not tied to phase
+    	Xpp3Dom bootstrapPropsConfig = config.getChildren("bootstrapProperties")[0];
```

Comment:
We need to check if it actually has `bootstrapProperties`, otherwise this would cause `Array index out of range`

Reply:
Done